tests/integration-tests/Apple/Simulator.Scouting.Tests.proj

```diff
@@ -0,0 +1,34 @@
+<Project DefaultTargets="Test">
+  <Import Project="../Helix.SDK.configuration.props"/>
+
+  <ItemGroup>
+    <HelixTargetQueue Include="osx.1200.amd64.scouting.open"/>
+
+    <!-- apple test / ios-simulator-64 -->
+    <XHarnessAppleProject Include="$(MSBuildThisFileDirectory)\TestAppBundle.proj">
+      <AdditionalProperties>TestTarget=ios-simulator-64;TestAppBundleName=System.Numerics.Vectors.Tests.app</AdditionalProperties>
+    </XHarnessAppleProject>
+
+    <!-- apple run / ios-simlator-64 -->
+    <XHarnessAppleProject Include="$(MSBuildThisFileDirectory)\TestAppBundle.proj">
+      <AdditionalProperties>TestTarget=ios-simulator-64;TestAppBundleName=iOS.Simulator.PInvoke.Test.app;IncludesTestRunner=false;ExpectedExitCode=42</AdditionalProperties>
+    </XHarnessAppleProject>
+
+    <!-- apple test / maccatalyst -->
+    <XHarnessAppleProject Include="$(MSBuildThisFileDirectory)\TestAppBundle.proj">
+      <AdditionalProperties>TestTarget=maccatalyst;TestAppBundleName=System.AppContext.Tests.app</AdditionalProperties>
+    </XHarnessAppleProject>
+
+    <!-- apple run / maccatalyst -->
+    <XHarnessAppleProject Include="$(MSBuildThisFileDirectory)\TestAppBundle.proj">
+      <AdditionalProperties>TestTarget=maccatalyst;TestAppBundleName=iOS.Simulator.PInvoke.Test.app;IncludesTestRunner=false;ExpectedExitCode=42</AdditionalProperties>
+    </XHarnessAppleProject>
+
+    <!-- apple test / tvos-simulator -->
+    <XHarnessAppleProject Include="$(MSBuildThisFileDirectory)\TestAppBundle.proj">
+      <AdditionalProperties>TestTarget=tvos-simulator;TestAppBundleName=Microsoft.Extensions.Configuration.Ini.Tests.app</AdditionalProperties>
```

Comment:
LGTM, just a question - the <XHarnessAppleProject/> node differs only in the AdditionalProperties, otherwise it's the same (the Include attribute). It's not a big deal in this change, but is it possible to write this more concisely?

Reply:
I don't think there is a better way to make this even more concise - we have to call the `TestAppBundle.proj` with different parameters